### DISCUSSION TITLE
"step certificate create" does not update the database ?

### DISCUSSION THREAD

**Discussion by tmartincpp** [smallstep/certificates#1334] [General] [answered]
Hello,

When I'm using "step certificate create" to create a certificate (using JWK then), step does not seems to insert the certificate in the database.
In the contrary database is updated when I'm using ACME provisioner.

To see what is in the DB I'm using [listcerts.go](https://gist.github.com/maraino/4dcb64cb051b17ef6d421892cb4e55a8).

Am I missing something (almost certain I am...) ?

Thanks!

Example of command used : 
```
step certificate create imap.test.com imap.test.com.20230329.crt imap.test.com.20230329.key --san imap.test.com --san smtp.test.com --ca .step/certs/intermediate_ca.crt --ca-key .step/secrets/intermediate_ca_key.dec --no-password --insecure --not-after 87660h
```

**Comment by maraino** [comment, marked as answer]
`step certificate create` does not talk to `step-ca`. That command is to create certificates 100% offline, and it doesn't use a JWK provisioner at all. It signs the certificates with the `--ca` and `--ca-key` you provide. 

To generate X.509 certificates on the CA you need to use `step ca certificate` or `step ca sign`.

**Comment by tmartincpp** [comment]
Thank you, I knew I was missing something, my bad.
